chirp/drivers/radtel_rt493.py

```diff
@@ -0,0 +1,1021 @@
+            mem.empty = True
+            return
+
+        mem.freq = int(_mem.rxfreq) * 10
+        chirp_common.split_to_offset(mem, int(_mem.rxfreq) * 10,
+                                     int(_mem.txfreq) * 10)
+        txtone = self._decode_qt_dqt(_mem.txtone.get_raw())
+        rxtone = self._decode_qt_dqt(_mem.rxtone.get_raw())
+        chirp_common.split_tone_decode(mem, txtone, rxtone)
+
+        # main_settings_byte = _mem.settings.get_raw()
+        # main_settings = _decode_channel_settings(main_settings_byte)
+        mem.mode = "WFM" if int(_mem.wn_mode) else "NFM"
+
+        power_level = self.get_power_level(mem.number)
+        mem.power = self.POWER_LEVELS[0 if power_level else 1]
+
+        scan_add = self.get_scan_add_status(mem.number)
+        if not scan_add:
+            mem.skip = "S"
+
+        mem.extra = RadioSettingGroup("Extra", "extra")
+
+        rs = RadioSetting(
+            "Compander", "Compander",
+            RadioSettingValueBoolean(_mem.compander))
+        mem.extra.append(rs)
+
+        rs = RadioSetting(
+            "Scramble Mode", "Scramble Mode",
+            RadioSettingValueList(
+                ["OFF", "1", "2", "3", "4", "5", "6", "7"],
+                current_index=_mem.scramble
+            ))
+        mem.extra.append(rs)
+
+        rs = RadioSetting(
+            "Special QT/DQT", "Special QT/DQT",
+            RadioSettingValueList(
+                ["OFF", "A", "B"],
+                current_index=0 if _mem.special_qt_dqt == 0x00
+                else (1 if _mem.special_qt_dqt == 0x01 else 2)
+            ))
+        mem.extra.append(rs)
+
+        LOG.debug("WN Mode: %i, Compander: %i, Scramble: %i, Special QT/DQT: \
+                  %i" % (int(_mem.wn_mode), int(_mem.compander),
+                         int(_mem.scramble), int(_mem.special_qt_dqt)))
+
+        learn_code = self.get_learn_code_status(mem.number)
+        rs = RadioSetting(
+            "Learn Code", "Learn Code",
+            RadioSettingValueBoolean(learn_code))
+        mem.extra.append(rs)
+
+        special_code = self.get_special_code(mem.number)
+        rs = RadioSetting(
+            "Special Code", "Special Code",
+            RadioSettingValueString(
+                8, 8, special_code, True, CHARSET_HEX, "0"))
+        mem.extra.append(rs)
+
+    def get_memory(self, number):
+        if number <= 0 or number > self._upper:
+            raise errors.InvalidMemoryLocation(
+                "Number must be between 1 and %i (included)" % self._upper)
+
+        _mem = self._memobj.memory[number-1]
+
+        mem = chirp_common.Memory()
+        mem.number = number
+
+        self.decode_mem_blob(_mem, mem)
+
+        return mem
+
+    def _encode_qt_dqt(self, tone_mode, tone_value=None, tone_polarity=None):
+        if tone_mode == "Tone" or tone_mode == "TSQL":
+            assert tone_value is not None
+            return int(tone_value * 10).to_bytes(2, "little")  # Encode as Hz
```

Comment:
Please do not use `int.to_bytes()` like this. The bitwise field should handle the endianess and conversion. And please don't do your own formatting of low-level memory data except in places where it's entirely avoidable.